lib/scrabble.rb

```diff
@@ -1,9 +1,18 @@
 class Scrabble
+  attr_accessor :word
+
+  def initialize
+    @word = word
+  end
 
   def score(word)
-    1
+    var = word.chars.map do |x|
```

Comment:
Like the high level logic here: split the characters into an array, swap them out for values, add those values up. It makes it all seem very straightforward.

Consider breaking this down into smaller methods that you then call from this top level method. Would then allow you to do some unit testing, and (if named appropriately) to make this logic even more clear.